### ISSUE TITLE
Readme.md - does not show all contributions

### ISSUE THREAD

**Issue by Ebazhanov** [Ebazhanov/linkedin-skill-assessments-quizzes#4954]
For some reason browser does not show what is exactly inside Readme.md but only the bottom part
![image](https://user-images.githubusercontent.com/13170022/196977192-de7bd8ae-86e0-4fd8-a0ec-e7fd4d3a1829.png)

**Workaround:** 
- open this file in a new tab https://github.com/Ebazhanov/linkedin-skill-assessments-quizzes/blob/main/README.md
- or use https://ebazhanov.github.io/linkedin-skill-assessments-quizzes/

While experimenting, I found that if I cut a few lines <10 from the Readme.md file, then in total we will have less and the rest is visible. So, I think that we have used the entire range of visibility. But why then is it displayed when following the [link](https://github.com/Ebazhanov/linkedin-skill-assessments-quizzes/blob/main/README.md)?

**Comment by antra0497** [comment]
Did you already solved this??
![image](https://user-images.githubusercontent.com/25953832/197350317-9de0905a-02a7-4fdb-b669-c14ed06b67ad.png)
It looks fine on my end.

**Comment by Ebazhanov** [comment]
yes still have problem to see, even if I use different browsers (private modes etc)

The issue is only happens if you scroll to the top from this direct URL https://github.com/Ebazhanov/linkedin-skill-assessments-quizzes

thanks for feedback @antra0497, I would be happy if it is not affected anyone else except me )))

**Comment by auchynnikau** [comment]
@Ebazhanov also can't reproduce this issue:

<img width="671" alt="image" src="https://user-images.githubusercontent.com/33329898/197355931-e4dc63c0-d0e9-4c10-9800-c021381b1a10.png">

<img width="952" alt="image" src="https://user-images.githubusercontent.com/33329898/197355994-356b5e17-f6b4-4839-b258-8c4d08cab2fa.png">

**Comment by antra0497** [comment]
@Ebazhanov  with normal viewing _(when logged in)_ everything is fine over both windows and mac browsers. 

I can see two differences in incognito mode: firstly, the list of contributors seems different, and secondly, the last section of contribution over time is not visible. 

As the list of contributors is growing why not change it to some other format??

**Comment by Ebazhanov** [comment]
Thanks for analysing guys, I am open for any suggestion!
